### PR TITLE
Proto defs for aggregates

### PR DESCRIPTION
**Pull request by Dmytry-Dyachenko**
Add protobuf definition for models:

```
- Book
- Inventory
- InventoryItem
- Reservation
- Lost
```
Edit comments in all `.proto` files.

### CODE REVIEW

**Review by PaulAgeyev — APPROVED** [review]
LGTM

**Review by alexanderkarpetc — COMMENTED** [review]
Update project and make whitespaces in commands and event please.

File: model/src/main/proto/javaclasses/exlibris/values.proto [hunk 1]
```diff
@@ -48,42 +49,50 @@ message BookDetails {
     repeated Category categories = 5;
 }
 
-// Message that consist of URL for book cover.
+// URL for book cover.
 //
 message BookCoverURL {
```
Comment (alexanderkarpetc) [inline model/src/main/proto/javaclasses/exlibris/values.proto:54]: Change to camel.

File: model/src/main/proto/javaclasses/exlibris/values.proto [hunk 2]
```diff
@@ -48,42 +49,50 @@ message BookDetails {
...
+// The name of category.
 //
 message Category {
 
     string category = 1;
 }
 
+// Unique mark for book.
+//
 message RFID {
```
Comment (alexanderkarpetc) [inline model/src/main/proto/javaclasses/exlibris/values.proto:89]: Change to camel

File: model/src/main/proto/javaclasses/exlibris/values.proto [hunk 3]
```diff
@@ -48,42 +49,50 @@ message BookDetails {
...
 
     string rfid = 1;
 }
-// Message that consist of user’s email.
+
+// The string representation of user’s email address in the next type:
+// <Name>.<Surname>@teamdev.com
 //
 message EmailAddress {
```
Comment (alexanderkarpetc) [inline model/src/main/proto/javaclasses/exlibris/values.proto:97]: Change to proto emailType

**Pull request merged**